service/src/test/java/bio/terra/pipelines/service/PipelinesServiceTest.java

```diff
@@ -235,7 +243,10 @@ private static Stream<Arguments> badWdlMethodVersions() {
         arguments("blah.3.2"),
         arguments("0.2"),
         arguments("1.bhmm.2"),
-        arguments("1.4.ok"));
+        arguments("1.4.ok"),
+        arguments(
+            "ImputationBeagle_development_0.0.0"), // missing a "v" before the semantic version
+        arguments("hiiv.1.4"));
```

Comment:
do we expect "v0.1.2" to work? can we put it in a test?

Reply:
we do expect it to work, its part of the same regex group as the current test.  do you want one specifically with this string?